src/java/org/apache/cassandra/db/marshal/DecimalType.java

```diff
@@ -50,6 +50,10 @@
     private static final int POSITIVE_DECIMAL_HEADER_MASK = 0x80;
     private static final int NEGATIVE_DECIMAL_HEADER_MASK = 0x00;
     private static final int DECIMAL_EXPONENT_LENGTH_HEADER_MASK = 0x40;
+    private static final byte DECIMAL_LAST_BYTE = (byte) 0x00;
+    private static final BigInteger HUNDRED = BigInteger.valueOf(100);
+
+    private static final ByteBuffer ZERO_ARRAY = instance.decompose(BigDecimal.ZERO);
```

Comment:
nit: why `ZERO_ARRAY` when it is a buffer?

Reply:
Done